executor/insert.go

```diff
@@ -315,6 +315,9 @@ func (e *InsertExec) Next(ctx context.Context, req *chunk.Chunk) error {
 
 // Close implements the Executor Close interface.
 func (e *InsertExec) Close() error {
+	if e.memTracker != nil {
+		defer e.memTracker.Detach()
```

Comment:
Just curious: is it necessary?

Reply:
updated.

Reply:
Is it possible to use more memory after the executor is closed? I didn't get it why we need to detach it.